lib/ecco/row_event_listener.rb

```diff
@@ -1,35 +1,21 @@
 require "ecco/event_listener"
 require "ecco/row_event"
+require_relative "row_type"
```

Comment:
this seems to break the pattern for requires, use:

```suggestion
require "ecco/row_type"
```

Reply:
require pattern aligned: 02f8b78380454419d6fc90559eff9079c17e146d

Reply:
Great, consider using "Commit suggestion" next time though :)

<img width="240" alt="Screenshot 2020-11-03 at 08 06 29" src="https://user-images.githubusercontent.com/2550510/97957443-80a76800-1dab-11eb-812c-4ef949736ca6.png">

Reply:
Ahh sorry!

Reply:
hehe this came back?

Reply:
Huh, that is very odd! I'm not really sure how this happened, even when looking through the commits 😅